eventbridge-pipes-ddbstream-sfn/template.yaml

```diff
@@ -0,0 +1,147 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Template to launch StepFunctions with DynamoDB Stream
+
+Parameters:
+  NationalTeam:
+    Description: National Team Name
+    Type: String
+    Default: Argentina
+
+Resources:
+  # DynamoDB Table Creation with Stream Enabled
+  DynamoDBWCTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: WorldCupTable
+      AttributeDefinitions:
+        - AttributeName: PlayerName
+          AttributeType: S
+      KeySchema:
+        - AttributeName: PlayerName
+          KeyType: HASH
+      ProvisionedThroughput:
+        ReadCapacityUnits: 5
```

Comment:
Do you see any issue with setting read and write capacity to 1 (or changing it to on-demand)?

Reply:
No @bfreiberg, I do not see any issue. Updated read and write capacity to 1.